src/providers/gpx/qgsgpxfeatureiterator.cpp

```diff
@@ -497,8 +497,8 @@ QgsGeometry *QgsGPXFeatureIterator::readTrackGeometry( const QgsTrack &trk )
 QgsGPXFeatureSource::QgsGPXFeatureSource( const QgsGPXProvider *p )
   : mFileName( p->mFileName )
   , mFeatureType( p->mFeatureType )
-  , indexToAttr( p->indexToAttr )
-  , mFields( p->attributeFields )
+  , indexToAttr( p->mIndexToAttr )
```

Comment:
Not directly related to this PR, but a follow up cleanup that'd be nice is the renaming of QgsGPXFeatureSource private members indexToAttr and data to mIndexToAttr and mData too.

Reply:
done

Reply:
@nyalldawson , thanks.